### DISCUSSION TITLE
Wait, what? Future milestone issue about GCI leads to a 404 ...

### DISCUSSION THREAD

**Discussion by God-damnit-all** [PowerShell/PowerShell#21248] [General]
On [this page](https://github.com/PowerShell/PowerShell/milestone/48) for PowerShell's future milestones, issue #8509 is listed, titled: `Get-ChildItem takes a very long time`.

Clicking on it leads to a 404.

What's going on here?

**Comment by MartinGC94** [comment]
I would assume it's been made private for some reason, probably by mistake.
